pkg/deploy/submariner.go

```diff
@@ -126,6 +130,10 @@ func populateSubmarinerSpec(options *SubmarinerOptions, brokerInfo *broker.Info,
 		submarinerSpec.GlobalCIDR = netconfig.GlobalCIDR
 	}
 
+	if clustersetConfig.ClustersetIPCIDR != "" {
+		submarinerSpec.ClustersetIPCIDR = clustersetConfig.ClustersetIPCIDR
+	}
```

Comment:
We can just initialize above regardless if empty.

```suggestion
```

Reply:
Ah right, forgot that this is `omitifempty`.